composer.json

```
@@ -6,20 +6,28 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
-        "drupal/core": "^8.9 || ^9.1"
+        "php": ">=7.4",
+        "drupal/core": "^9.2"
     },
     "require-dev": {
         "composer/installers": "^1.5",
-        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
+        "drupal/core-composer-scaffold": "^9.2",
         "drupal/config_devel": "^1.2",
         "drush/drush": "^10.3",
-        "openeuropa/code-review": "^1.7",
-        "openeuropa/drupal-core-require-dev": "^8.9 || ^9.1",
-        "openeuropa/oe_multilingual": "^1.5",
-        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "drupal/core-dev": "^9.2",
+        "drupal/ctools": "^3.7",
+        "drupal/token": "^1.10",
+        "egulias/email-validator": "^2.1.22 || ^3.0",
+        "openeuropa/oe_multilingual": "^1.10",
+        "openeuropa/code-review": "^1.6 || ^2.0",
```

```suggestion
        "openeuropa/code-review": "^2.0",
```
Let's switch to version 2 since the dependency phpro/grumphp uses different formats of grumphp.yml in versions 0.17 and 1. Thus we will decrease possible issues with grumphp.yml on low version of openeuropa/code-review.